README.md

```diff
@@ -97,17 +97,29 @@ By default, the document will attempt to be decoded as utf8. This is the default
 
 ### configure(options: IOptions)
 
-Configures linez to use the supplied options. Currently, only the newlines property is available, where you can specify any number of newline character sequences.
+Set the globle configures linez to use the supplied options. you can specify any number of newline character sequences in newlines property, and you can specify any number of code block options in block property.
 
 ```js
 linez.configure({
-  newlines: ['\n', '\r\n', '\r', '\u000B']
+  newlines: ['\n', '\r\n', '\r', '\u000B'],
+  blocks: [
+    {
+      type: 'multilineString',
+      start: /[^\\]".*\\\s*$/,
+      end: /[^\\]";?\s*$/,
+    },
+    {
```

Comment:
I'm just wondering why it's important to capture a multi-line string, for the purposes of this package. I'm afraid of "doing too much" with this package and I want to keep it as simple as possible, unless there's a very specific reason to do otherwise. What goal are we trying to achieve here?